docs/config.json

```
@@ -1710,6 +1710,16 @@
       "destination": "/admin-guides/access-controls/access-request-plugins/ssh-approval-slack/",
       "permanent": true
     },
+    {
+      "source": "/access-controls/access-request-plugins/datadog-hosted/",
+      "destination": "/admin-guides/access-controls/access-request-plugins/datadog-hosted/",
+      "permanent": true
+    },
+    {
+      "source": "/access-controls/access-request-plugins/ssh-approval-datadog/",
+      "destination": "/admin-guides/access-controls/access-request-plugins/ssh-approval-datadog/",
```

Not sure why these have "ssh" in the page names, I'm guessing it's an artifact of old guides or something, I'd remove it as it doesn't make sense.
```suggestion
      "source": "/access-controls/access-request-plugins/datadog-self-hosted/",
      "destination": "/admin-guides/access-controls/access-request-plugins/datadog-self-hosted/",
```